bin/lang-js/src/function.ts

```diff
@@ -19,6 +19,7 @@ import { Debugger } from "./debug.ts";
 import { transpile } from "jsr:@deno/emit";
 import { Debug } from "./debug.ts";
 import * as _worker from "./worker.js";
+import { bundleCode } from "./transpile.ts";
```

Comment:
What's going on here exactly?

Reply:
We need to go from ts -> js so we can build it out as a function to run in the web worker